libs/sdk-ui-kit/src/Dialog/EmbedInsightDialog/EmbedInsightDialogBase/components/HeightSetting.tsx

```diff
@@ -0,0 +1,91 @@
+// (C) 2022 GoodData Corporation
+import React, { useCallback } from "react";
+import { SingleSelectListItem } from "../../../../List";
+import { Dropdown, DropdownButton, DropdownList } from "../../../../Dropdown";
+import { NumericInput } from "./NumericInput";
+import { DEFAULT_UNIT, UNITS, UnitsType } from "../types";
+import { getDefaultHeightForEmbedCodeByUnit } from "../../utils";
+
+/**
+ * @internal
+ */
+export interface IHeightSettingProps {
+    value?: string;
+    unit?: UnitsType;
+    onValueChange: (value: string, unit: UnitsType) => void;
+}
+
+/**
+ * @internal
+ */
+export const HeightSetting: React.VFC<IHeightSettingProps> = (props) => {
+    const { value, onValueChange, unit = DEFAULT_UNIT } = props;
+
+    const onChange = useCallback(
+        (val: string) => {
+            if (val != value) {
```

Comment:
Is the `!=` instead of `!==` intentional? If yes, please add a comment.

Reply:
Fixed